backend/controllers/adminController.js

```
@@ -11,13 +11,121 @@
+      return (
+        name.toLowerCase().includes(ql) ||
+        email.toLowerCase().includes(ql) ||
+        apptId.includes(ql) ||
+        (l.fileUrl || '').toLowerCase().includes(ql)
+      );
+    });
+  }
+
+  // Build CSV
+  const headers = ['timestamp', 'generatedById', 'generatedByName', 'generatedByEmail', 'appointmentId', 'appointmentDate', 'appointmentTime', 'fileUrl', 'ip', 'userAgent'];
+  const escape = (v) => '"' + (v === undefined || v === null ? '' : String(v).replace(/"/g, '""')) + '"';
```

Biome (2.1.2)

[error] Do not shadow the global "escape" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)